src/i18n/fi/draft.translation.json

```diff
@@ -42,5 +42,54 @@
     "settings": {
       "controls": "Säätimet"
     }
+  },
+  "profile": {
```

Comment:
These new keys are not yet sorted to alphabetic order.

Haven't tested this myself, but IDEA has something similar like VSCode and Vim to do that: https://plugins.jetbrains.com/plugin/11104-json-sorter